### PR TITLE
Move need of inplaceLibCopyTargets to top-level

### PR DESCRIPTION
**Pull request by izgzhen**

### CODE REVIEW

**Comment by izgzhen** [comment]
BTW, I suspect that we might have run out of CircleCI's plan quota (shrugging ...) I'll ask since we didn't receive any notification about it

File: src/Rules.hs [hunk 1]
```diff
@@ -29,6 +30,7 @@ allStages = [minBound ..]
 -- or Stage1Only targets
 topLevelTargets :: Rules ()
 topLevelTargets = action $ do
+    need inplaceLibCopyTargets
```
Comment (snowleopard) [inline src/Rules.hs]: `need` is a blocking operation, so it's better to put all targets together and do a single top-level `need` -- see the code below.

**Comment by snowleopard** [comment]
> BTW, I suspect that we might have run out of CircleCI's plan quota (shrugging ...) I'll ask since we didn't receive any notification about it

Yes, please. I haven't received anything from them.

**Comment by izgzhen** [comment]
Updated!

**Comment by snowleopard** [comment]
@izgzhen Thank you, merged!

**Pull request merged**